.github/QA_TEMPLATE/certification.md

```diff
@@ -0,0 +1,96 @@
+### Certification Documentation For [TITLE OF THE TASK/ISSUE/PROJECT/MILESTONE]  
+### Leader  
+[NAME: TITLE]  
+### Date(s)  
+01/01/1970  
+### Participants
+- [NAME:TITLE]
+- [NAME:TITLE]
+- [NAME:TITLE]
+## Survey
+### Category
+- [ ] Infrastructure  
+- [ ] Protocol/Blockchain Dev  
+- [ ] App Solutions  
+
+### LEVEL
+- [ ] RC
+- [ ] STABLE
+
+## Certification
+### Summary
+**How is this deliverable certified?**
+
+[Detailed account of the Certification Process]
+
+**Certification Contract**
+
+- [ ] The certification of this deliverable ensures Stability Level as described in the handbook
+
+**Justification Statement**
+
+[Provide a detailed statement of satisfaction of the Certification Level requirements]
+
+### Alternatives Considered
```

Comment:
I would add a result's section. The idea would be to have the test results as evidence, and not leave it to just words. Good examples would be, providing test execution data (if applicable), test cases statuspassed/failed/skipped/blocked, test scenario status (passed/failed/skipped/blocked), etc.